src/java.net.http/share/classes/java/net/http/HttpResponse.java

```diff
@@ -1178,6 +1178,12 @@ public static BodySubscriber<Path> ofFile(Path file) {
          * the underlying HTTP connection to be closed and prevent it
          * from being reused for subsequent operations.
          *
+         * @implNote The read method of the default implementation returned
```

Comment:
Hello Darragh, should we instead reword this as:

> @implNote The {@code read} method of the {@code InputStream} returned by the default implementation of this method will throw an {@code IOException} with the {@link Thread#isInterrupted() thread interrupt status set} if the thread is interrupted while blocking on read. In that case, the request will also be cancelled and the {@code InputStream} will be closed.

Additionally, irrespective of whether or not we decide to reword this, this file will need an update for the copyright year.

Reply:
Agreed but it should be `{@linkplain Thread#isInterrupted() thread interrupt status set}`